src/Chapter1.hs

```diff
@@ -619,9 +628,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n = last(div n 10) + last n 
+  where
+     last :: Int -> Int
+     last n = mod (abs n) (10 * div (abs n) 10)
```

Comment:
This function has the same issue as above `lastDigit` one. Once that is implemented, then you can just reuse it here instead of `last` and your solution would work perfectly! 👍🏼